utils/docker/Dockerfile.centos.7

```
@@ -205,6 +200,8 @@ RUN echo -e "<settings>\n\
         </mirrors>\n\
 </settings>" > /home/daos/.m2/settings.xml
 
+ARG DAOS_JAVA_BUILD=$DAOS_BUILD
+
 RUN [ "$DAOS_JAVA_BUILD" != "yes" ] || \
```

Use the more natural boolean logic available in the Bourne shell, and avoid double negatives:
```suggestion
RUN "$DAOS_JAVA_BUILD" && \
```